bot/exts/utils/snekbox.py

```diff
@@ -344,8 +344,14 @@ async def send_job(
                 log.trace("Formatting output...")
                 output, paste_link = await self.format_output(results["stdout"])
 
+            warning_message = ""
+            # 33 is the length of the error message. It is done to make sure the last line of output contains
+            # the error and the error is not manually printed by the author with a syntax error.
+            if "EOFError: EOF when reading a line" in output[-33:] and results['returncode'] == 1:
```

Comment:
should also sanity check the user had "input" in their code, IMO.

Reply:
Or the message should be altered to include stdin.

Reply:
As mentioned in the PR description, the docs say EOFError is specific to `input`, so i'm not sure what use case this would catch.

Reply:
This thought didn't cross my mind earlier what if someone raises the `EOFError`

Reply:
This thought didn't cross my mind earlier what if someone raises the `EOFError`, we should probably add a check for this.

Reply:
I don't think we need to add a check, I don't know why someone would raise the error manually, and even if they do the warning isn't incorrect, just unnecessary.